code/script.js

```diff
@@ -1,47 +1,181 @@
+    <button id="cocktailBtn">Cocktail bar</button>
+  `
+  document.getElementById('nightclubBtn')
+  .addEventListener('click', () => nextStep('Nightclub'))
+  document.getElementById('cocktailBtn')
+  .addEventListener('click', () => nextStep('Cocktail bar'))
+} 
+
+const showVibe = (place) => {
+  answerNumber++
+
+  if (place === 'Nightclub') {
+    botAnswer(`Which club?`)
+
+    inputWrapper.innerHTML = `
+    <select class="dropdown" id="select">
+      <option class="dropdown-item" value="" selected disabled>Pick your place! &#11015;</option>
+      <option class="dropdown-item" value="Berghain">Berghain</option>
+      <option class="dropdown-item" value="Studio 54">Studio 54</option>
+      <option class="dropdown-item" value="Nikki Beach">Nikki Beach</option>
+      <option class="dropdown-item" value="Cavalli Club">Cavalli Club</option>
+    </select>
+  `
+  } else if (place === "Cocktail bar") {
+    botAnswer(`What's your favourite cocktail?`)
+
+    inputWrapper.innerHTML = `
+    <select class="dropdown" id="select">
+      <option value="" selected disabled>Choose your poision &#11015;</option>
+      <option value="Cosmopolitan">Cosmopolitan</option>
+      <option value="Old fashioned">Old fashioned</option>
+      <option value="Moscow Mule">Moscow Mule</option>
+      <option value="Mojito">Mojito</option>
+    </select>
+  `
+  }
+  const outfit = document.getElementById('select')
+  outfit.addEventListener('change', () => nextStep(outfit.value))
 }
 
-// Set up your eventlisteners here
+const showOutfit = (outfit) => {
+  answerNumber++
+  botAnswer(`I got the perfect outfit for you! Party on!`)
+
+  const showFinalMessage = (gifSrc) => {
+    chat.innerHTML += `
+      <section class="bot-msg">
+        <img class="profile-pic" src="assets/bot-img.jpeg" alt="Bot" /> 
+        <div class="bubble bot-bubble final">
+          <img class="outfit-gif" src="${gifSrc}"/>
+        </div> 
+      </section>
+    `
+    //Allows the gif to scroll chat to bottom after gif is loaded
+    document.getElementsByClassName("outfit-gif")[0].onload = () =>
+      (chat.scrollTop = chat.scrollHeight)
+      console.log(chat.scrollTop, "scrollTop")
+}
+
+  if (outfit === 'Berghain') {
+    showFinalMessage("assets/berghain.gif")
+  }
+
+  else if (outfit === "Studio 54") {
+    showFinalMessage("assets/studio54.gif")
+  }
+
+  else if (outfit === "Nikki Beach") {
+    showFinalMessage("assets/nikkibeach.gif")
+  }
+
+  else if (outfit === "Cavalli Club") {
+    showFinalMessage("assets/cavalliclub.gif")
+  }
+
+  else if (outfit === "Cosmopolitan") {
+    showFinalMessage("assets/cocktail.gif")
+  }
+
+  else if (outfit === "Old fashioned") {
+    showFinalMessage("assets/oldfashioned.gif")
+  }
+
+  else if (outfit === "Moscow Mule") {
+    showFinalMessage("assets/mosow.gif")
+  }
+
+  else if (outfit === "Mojito") {
+    showFinalMessage("assets/mojito.gif")
+  }
+  inputWrapper.innerHTML = ""
+}
```

Comment:
This is a very long line of `if/else/if...`. In situations like this, a better way to structure your flow would be a `switch` statement. You can read more about it here: https://www.w3schools.com/js/js_switch.asp 🤓